tests/ballerina-test/src/test/java/org/ballerinalang/test/net/grpc/BalGenToolTest.java

```diff
@@ -126,19 +127,60 @@ public void testServerStreamingHelloWorld() throws IllegalAccessException,
         Path destFileRoot = resourceDir.resolve(Paths.get("protoFiles/helloWorldServerStreaming.gen.pb.bal"));
         removePackage(sourceFileRoot.toString(), destFileRoot.toString());
         CompileResult compileResult = BCompileUtil.compile(destFileRoot.toString());
-        Assert.assertNotNull(compileResult.getProgFile().getPackageInfo(".")
+        Assert.assertNotNull(compileResult.getProgFile().getPackageInfo(PACKAGE_NAME)
                         .getStructInfo("helloWorldServerStreamingClient"),
                 "Connector not found.");
-        Assert.assertNotNull(compileResult.getProgFile().getPackageInfo(".")
+        Assert.assertNotNull(compileResult.getProgFile().getPackageInfo(PACKAGE_NAME)
                         .getStructInfo("helloWorldServerStreamingStub"),
                 "Connector not found.");
-        Assert.assertNotNull(compileResult.getProgFile().getPackageInfo(".")
+        Assert.assertNotNull(compileResult.getProgFile().getPackageInfo(PACKAGE_NAME)
                         .getFunctionInfo("helloWorldServerStreamingStub.lotsOfReplies"),
                 "Connector not found.");
         String protoExeName = "protoc-" + OSDetector.getDetectedClassifier() + ".exe";
         BalFileGenerationUtils.delete(new File(protoExeName));
     }
     
+    @Test
+    public void testStandardDataTypes() throws IllegalAccessException,
+            ClassNotFoundException, InstantiationException, IOException {
+        Class<?> grpcCmd = Class.forName("org.ballerinalang.protobuf.cmd.GrpcCmd");
+        GrpcCmd grpcCmd1 = (GrpcCmd) grpcCmd.newInstance();
+        Path sourcePath = Paths.get("protoFiles");
+        Path sourceRoot = resourceDir.resolve(sourcePath);
+        Path protoPath = Paths.get("protoFiles/helloWorldString.proto");
+        Path protoRoot = resourceDir.resolve(protoPath);
+        grpcCmd1.setBalOutPath(sourceRoot.toString());
+        grpcCmd1.setProtoPath(protoRoot.toString());
+        grpcCmd1.execute();
+        Path sourceFileRoot = resourceDir.resolve(Paths.get("protoFiles/helloWorld.pb.bal"));
+        Path destFileRoot = resourceDir.resolve(Paths.get("protoFiles/helloWorld.gen.pb.bal"));
+        removePackage(sourceFileRoot.toString(), destFileRoot.toString());
+        CompileResult compileResult = BCompileUtil.compile(destFileRoot.toString());
+        Assert.assertNotNull(compileResult.getProgFile().getPackageInfo(PACKAGE_NAME)
+                        .getStructInfo("helloWorldClient"),
+                "Connector not found.");
```

Comment:
This line can be moved to the previous line. Similar fix can be done for below lines.

Reply:
Fixed in f2b7cd26cb71721658fbc018e48d1aecaddaecd2